app/src/main/java/com/kamron/pogoiv/pokeflycomponents/GoIVNotificationManager.java

```diff
@@ -139,10 +144,31 @@ public void showRunningNotification() {
                 .setCategory(NotificationCompat.CATEGORY_PROGRESS)
                 .setContent(contentView)
                 .setCustomBigContentView(contentBigView)
-                .setOngoing(true)
-                .build();
+                .setOngoing(true);
+
+        NotificationManager notificationManager =
+                (NotificationManager) pokefly.getSystemService(Context.NOTIFICATION_SERVICE);
+
+        initNotificationChannel(notificationManager);
 
-        pokefly.startForeground(NOTIFICATION_REQ_CODE, notification);
+        pokefly.startForeground(NOTIFICATION_REQ_CODE, notification.build());
+    }
+
+    /**
+     * The class which create channel notification for oreo.
+     */
+    private void initNotificationChannel(NotificationManager notificationManager) {
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O && notificationManager
+                .getNotificationChannel(NOTIFICATION_CHANNEL_ID) == null) {
+            // Create notification channel
+            NotificationChannel channel = new NotificationChannel(NOTIFICATION_CHANNEL_ID,
+                    "GoIV", NotificationManager.IMPORTANCE_LOW);
```

Comment:
You should put a translatable string here instead of "GoIV".
Also, the channel name must be more explicative.
I'd set it to "GoIV status" or something similar, so the user knows that disabling this channel in settings will cause the GoIV status notification (running / paused) to disappear.

Reply:
the user see this in the notification settings

![captura de pantalla 2017-10-20 a las 14 47 57](https://user-images.githubusercontent.com/23555838/31821565-dbd0087a-b5a5-11e7-8977-5e612ba33a69.png)

Do you still want to change it to status ?